--- /usr/local/lib/python2.3/site-packages/mmpython/audio/eyed3info.py.org	Sat Aug 30 11:36:22 2003
+++ /usr/local/lib/python2.3/site-packages/mmpython/audio/eyed3info.py	Mon Jan 19 05:20:49 2004
@@ -145,6 +145,8 @@
                      setattr(self, MP3_INFO_TABLE[k], id3.tag.frames[k][0].text)
             if id3.tag.getYear():
                self.date = id3.tag.getYear()
+            if id3.tag.getGenre():
+               self.genre = id3.tag.getGenre().name
             tab = {}
             for f in id3.tag.frames:
                 if f.__class__ is eyeD3_frames.TextFrame:                
